index.html

```diff
@@ -229,6 +207,27 @@ <h2>
               </ul>
           </div>
         </div>
+
+        <section class="contact-me">
+          <h3 class="contact-me-header">
+           I'm always interested in hearing about new projects,
+            so if you'd like to chat please get in touch.
+          </h3>
+          <form action="#" method="post" class="form-contact-me">
+            <input type="text"  name="username" placeholder="Full name" id="name" required>
+            <input type="email" name="email" placeholder="Full name" id="email" required>
+            <textarea name="comment" id="text-area" cols="30" rows="10">Enter text here</textarea>
+            <button type="submit" class="contact-me-btn1">Get in touch</button>
+          </form>
+          <div class="header-icons footer-icons">
+           <ul>
+               <li><a href="#"><i class="fa-brands fa-github" aria-hidden="true"></i></a></li>
+               <li><a href="#"><i class="fa-brands fa-linkedin" aria-hidden="true"></i></a></li>
+               <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+               <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>     
```

Comment:
- [ ] Please increase the size of these social media icons and make sure it matches with the Figma design

### Your design

![image](https://user-images.githubusercontent.com/83803414/165283468-b3f87aea-81d0-4fd5-9d0f-d4574049f94a.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/165283518-b32c4a7f-0aed-4028-b1d0-a97853e6a11a.png)